scenes/game/game.gd

```diff
@@ -16,7 +19,28 @@ func _on_xp_timer_timeout() -> void:
 	_update_ui()
 	GameStorage.save_progress()
 
+func update_tree_model():
+	for child in tree_parent.get_children():
+		child.queue_free()
+	
+	# Load and instance the new tree
+	var tree = TreeLS.get_select_tree() 
+	var tree_scene = load(tree["asset"])
+	if tree_scene:
+		var tree_instance = tree_scene.instantiate()
+		tree_parent.add_child(tree_instance)
+		match tree["name"]: #All assets have different sizes so we need to change its placing
+			"Tree 1":
+				tree_instance.position = Vector3(-7.745, 0, -10.357)
+			"Tree 2":
+				tree_instance.position = Vector3(50.982, 0, -9.077)
+			"Tree 3":
+				tree_instance.position = Vector3(33.712, 0, -9.771)
+		
```

Comment:
Please do not do this, create a godot scene for each tree and set the Mesh node position in that new scene